presidio-analyzer/presidio_analyzer/analyzer_loader.py

```diff
@@ -0,0 +1,72 @@
+import yaml
+import os
+import logging
+from typing import List
+
+from presidio_analyzer import AnalyzerEngine, RecognizerRegistry
+from presidio_analyzer.nlp_engine import NlpEngineProvider
+import presidio_analyzer.recognizer_registry
+
+logger = logging.getLogger("presidio-analyzer")
+
+
+class PresidioAnalyzerLoader:
+    """
+    Utility function for loading Presidio Analyzer.
+
+    Use this class to load presidio analyzer engine from a yaml file
+    """
+
+    @staticmethod
+    def load(conf_file: str) -> AnalyzerEngine:
```

Comment:
Not that there's anything wrong with this API, but we should consider having similar APIs across the package. in `NlpEngineProvider` the API is `NlpEngineProvider(conf_file).create_engine()`. Can we use something similar? I would also consider having the same name (`Provider`).

Reply:
Yeah I like this approach, deal